pkg/porter/testdata/cache/README.md

```diff
@@ -1,5 +1,5 @@
 # Cache Cheat Sheet
 
 * deislabs/kubekahn:1.0 -> 887e7e65e39277f8744bd00278760b06
-* getporter/mysql:v0.1.2 -> 972978d3b715212b783a70fdc3449584
-* getporter/wordpress:v0.1.2 -> bfa9f2b528338f9ac1bc9b8b984c3997
+* getporter/mysql:v0.1.3 -> 4f17468b1ad86cb50cd76d9b8148d249
```

Comment:
Thanks for updating my beloved cheat sheet!

Reply:
I backed out the changes in this directory now that we've placed the logic in PullBundle (and, hence, cached bundles wouldn't hit this case).  For another day!